### DISCUSSION TITLE
Styles | Web developer

### DISCUSSION THREAD

**Discussion by styles3544** [magbanum/Python-Community-Discussions#23] [Say hello]
Hey there,

I am Styles. I'm a curious enthusiast of ML/AI.

I know Python, HTML, CSS, Bootstrap, CAD software, and Git. I am currently working with web development skills and many web technologies. I'm also a python developer and loves to work with python. 

Visit my profile to know more about my work.

Profile: https://github.com/styles3544

**Comment by vivekvjyn** [comment]
Nice

**Comment by harshavardhanBOMMALATA** [comment]
Nice
